src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -15,13 +15,33 @@ namespace System.Device.Gpio.Drivers
     public class SysFsDriver : UnixDriver
     {
         private const string GpioBasePath = "/sys/class/gpio";
+        private const string GpioChip = "gpiochip";
+        private const string GpioLabel = "/label";
+        private const string GpioContoller = "pinctrl";
+        private const string GpioOffsetBase = "/base";
         private int _pollFileDescriptor = -1;
         private Thread _eventDetectionThread;
         private int _pinsToDetectEventsCount;
         private readonly static CancellationTokenSource s_eventThreadCancellationTokenSource = new CancellationTokenSource();
         private readonly List<int> _exportedPins = new List<int>();
         private readonly Dictionary<int, UnixDriverDevicePin> _devicePins = new Dictionary<int, UnixDriverDevicePin>();
         private readonly int _pollingTimeoutInMilliseconds = Convert.ToInt32(TimeSpan.FromMilliseconds(1).TotalMilliseconds);
+        private int _pinOffset = 0;
+
+        public SysFsDriver() {
```

Comment:
>SysFsDriver [](start = 15, length = 11)

what happen if we enumerated ass items under GpioBasePath and didn't find any? should we throw?

Reply:
not sure about that, i checked these path only on Raspberry pi with 32, 64bit installed, guess we could still try with the default pin numbering